addons/x_lib/functions/actors/fnc_allActors.sqf

```diff
@@ -23,18 +23,21 @@ See Also:
     - <ALIVE_fnc_sendActorMessage>
 
 Author:
-    Naught
+    Naught, dixon13
 ---------------------------------------------------------------------------- */
 
-private ["_group"];
-_group = [_this, 1, ["GROUP"], ALiVE_actors_mainGroup] call ALiVE_fnc_param;
-
-if (!isNil "_group") then
-{
+params [["_group", grpNull]];
+
+if (_group == grpNull) then {
+    _group = ALiVE_actors_mainGroup;
+    if (!isNil "_group") then {
+        units _group
+    } else {
+        LOG_WARNING("ALiVE_fnc_allActors", "Attempted to list actors before the main actor group was created!");
+        []
+    };
+} else {
     units _group
 }
-else
-{
-    LOG_WARNING("ALiVE_fnc_allActors", "Attempted to list actors before the main actor group was created!");
-    []
-};
+
+
```

Comment:
I'd like to suggest defining a variable at the top `private _result = []`, update that in the `if`s and return it at the end instead of semi-returning in the middle.